### PR TITLE
[doc][yba] Warning for disk encryption agent

### PR DESCRIPTION
**Pull request by ddhodge**
Warning for disk encryption agent
DOC-440

@netlify /preview/yugabyte-platform/security/enable-encryption-at-rest/#manage-universe-encryption-at-rest

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ The Linux OS and other software components on each database cluster node must me
 
 Depending on the [provider type](../../yba-overview/#provider-configurations) and permissions you grant, you may have to install all of these requirements manually, or YugabyteDB Anywhere (YBA) will install it all automatically.
 
+{{< warning title="Using disk encryption software with YugabyteDB" >}}
+If you are using third party disk encryption software, such as Vormetric or CipherTrust, the disk encryption service must be up and running on the node before starting any YugabyteDB services. If YugabyteDB processes start _before_ the encryption service, restarting an already encrypted node can result in data corruption.
+
+To avoid problems, pause the universe _before_ enabling or disabling the disk encryption service on universe nodes.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/_index.md:25]: Could be useful to mention how to do both - pausing a universe and disabling the disk encryption service

File: docs/content/preview/deploy/manual-deployment/install-software.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ Installing YugabyteDB involves completing prerequisites and downloading the Yuga
 
 {{% readfile "/preview/quick-start/include-prerequisites-linux.md" %}}
 
+### Using disk encryption software with YugabyteDB
+
+If you are using third party disk encryption software, such as Vormetric or CipherTrust, the disk encryption service must be up and running on the node before starting any YugabyteDB services. If YugabyteDB processes start _before_ the encryption service, restarting an already encrypted node can result in data corruption.
+
+To avoid issues, stop YugabyteDB services on the node _before_ enabling or disabling the disk encryption service.
```
Comment (aishwarya24) [inline docs/content/preview/deploy/manual-deployment/install-software.md:24]: same comment on adding details about how to do them

File: docs/content/preview/deploy/manual-deployment/start-masters.md [hunk 3]
```diff
@@ -32,7 +27,11 @@ This section covers deployment for a single region or data center in a multi-zon
...
 
-Run the yb-master server on each of the three nodes as follows. Note how multiple directories can be provided to the [`--fs_data_dirs`](../../../reference/configuration/yb-master/#fs-data-dirs) flag. Replace the [`--rpc_bind_addresses`](../../../reference/configuration/yb-master/#rpc-bind-addresses) value with the private IP address of the host as well as the set the `placement_cloud`,`placement_region` and `placement_zone` values appropriately. For single zone deployment, use the same value for the `placement_zone` flag.
+The number of nodes in a cluster running YB-Masters must equal the replication factor.
+
+Run the yb-master server on each of the three nodes as follows.
+
+Note how multiple directories can be provided to the [`--fs_data_dirs`](../../../reference/configuration/yb-master/#fs-data-dirs) flag. Replace the [`--rpc_bind_addresses`](../../../reference/configuration/yb-master/#rpc-bind-addresses) value with the private IP address of the host as well as the set the `placement_cloud`,`placement_region` and `placement_zone` values appropriately. For single zone deployment, use the same value for the `placement_zone` flag.
```
Comment (aishwarya24) [inline docs/content/preview/deploy/manual-deployment/start-masters.md:34]:
```suggestion
Note how multiple directories can be provided to the [`--fs_data_dirs`](../../../reference/configuration/yb-master/#fs-data-dirs) flag. Replace the [`--rpc_bind_addresses`](../../../reference/configuration/yb-master/#rpc-bind-addresses) value with the private IP address of the host, and set the `placement_cloud`, `placement_region`, and `placement_zone` values appropriately. For single zone deployment, use the same value for the `placement_zone` flag.
```

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/deploy/manual-deployment/start-tservers.md [hunk 4]
```diff
@@ -32,7 +28,11 @@ This section covers deployment for a single region or data center in a multi-zon
 
 ## Run YB-TServer with command line flags
 
-Run the yb-tserver server on each of the six nodes as follows. Note that all of the master addresses have to be provided using the `--tserver_master_addrs` flag. Replace the [`--rpc_bind_addresses`](../../../reference/configuration/yb-tserver/#rpc-bind-addresses) value with the private IP address of the host, and set the `placement_cloud`, `placement_region`, and `placement_zone` values appropriately. For single zone deployment, use the same value for the `--placement_zone` flag.
+The number of nodes in a cluster running YB-TServers must equal or exceed the replication factor in order for any table to get created successfully.
+
+Run the yb-tserver server on each of the six nodes as follows.
+
+Note that all of the master addresses have to be provided using the `--tserver_master_addrs` flag. Replace the [`--rpc_bind_addresses`](../../../reference/configuration/yb-tserver/#rpc-bind-addresses) value with the private IP address of the host, and set the `placement_cloud`, `placement_region`, and `placement_zone` values appropriately. For single zone deployment, use the same value for the `--placement_zone` flag.
```
Comment (ddhodge) [inline docs/content/preview/deploy/manual-deployment/start-tservers.md:35]:
```suggestion
You need to provide all of the master addresses using the `--tserver_master_addrs` flag. Replace the [`--rpc_bind_addresses`](../../../reference/configuration/yb-tserver/#rpc-bind-addresses) value with the private IP address of the host, and set the `placement_cloud`, `placement_region`, and `placement_zone` values appropriately. For single zone deployment, use the same value for the `--placement_zone` flag.
```

**Review by eugeneckim — APPROVED** [review]
Let's go!

**Pull request merged**